src/test/java/core/basesyntax/fileservise/WriteServiceImplTest.java

```diff
@@ -0,0 +1,47 @@
+package core.basesyntax.fileservise;
+
+import static org.junit.Assert.assertEquals;
+
+import java.io.BufferedWriter;
+import java.io.FileWriter;
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.util.List;
+import org.junit.Test;
+
+public class WriteServiceImplTest {
+    private static final String CORRECT_FILE_PATH = "src\\test\\resourcesTest\\report.csv";
+    private static final String INCORRECT_FILE_PATH = "src\\mein\\report.cvs";
+    private static final String report = "fruit,quantity" + System.lineSeparator()
+            + "banana,172" + System.lineSeparator()
+            + "apple,90" + System.lineSeparator();
+    private static final WriteServiceImpl writeService = new WriteServiceImpl();
+
+    @Test
+    public void write_reportToFile_Ok() throws IOException {
+        writeService.write(report, CORRECT_FILE_PATH);
+        List<String> expectedList;
+        try {
+            expectedList = Files.readAllLines(Path.of(CORRECT_FILE_PATH));
+        } catch (IOException e) {
+            throw new RuntimeException("Can't read file " + CORRECT_FILE_PATH, e);
+        }
+
+        BufferedWriter writer = new BufferedWriter(new FileWriter(CORRECT_FILE_PATH));
```

Comment:
we don't need all of this. read `actual` from file and make expected as a specific List you expect

Reply:
done